spark-bigquery-parent/pom.xml

```diff
@@ -244,6 +244,11 @@
                 <artifactId>arrow-vector</artifactId>
                 <version>${arrow.version}</version>
             </dependency>
+            <dependency>
```

Comment:
Is this a transitive dependency that we are fixing to a particular version?

Reply:
transitive from arrow, even in arrow 7 they haven't upgraded commons-compress, and we are getting security alerts on it

Reply:
Gotcha